crucible/src/Lang/Crucible/Simulator/ExecutionTree.hs

```diff
@@ -289,7 +313,14 @@ ppExceptionContext frames = PP.vcat (map pp (init frames))
 --   'PartialResult', then some of the computation paths that led to
 --   this result aborted for some reason, and the resulting value is
 --   only defined if the associated condition is true.
-data PartialResult sym ext (v :: Type)
+--
+--   Type parameters:
+--
+--   - @sym@: instance of 'Lang.Crucible.Backend.IsSymInterface'
+--   - @ext@: language extension, see "Lang.Crucible.CFG.Extension"
+--   - @v@: Type of the result of the compoutation
```

Comment:
```suggestion
--   - @v@: Type of the result of the computation
```

Reply:
This typo has inspired me to coin a new word:

> **compoutation**, n. Computation of a quantity of poutine.